main.nf

```diff
@@ -1839,6 +1838,13 @@ workflow.onComplete {
     c_purple = params.monochrome_logs ? '' : "\033[0;35m";
     c_green = params.monochrome_logs ? '' : "\033[0;32m";
     c_red = params.monochrome_logs ? '' : "\033[0;31m";
+
+    if (workflow.stats.ignoredCountFmt > 0 && workflow.success) {
+      log.info "${c_purple}Warning, pipeline completed, but with errored process(es) ${c_reset}"
```

Comment:
Maybe you want to get rid of this line, which is always reporting an error ...

Reply:
Yep. Ill do that when I bump-versions in the `dev` branch.